app/models/pglogical_subscription.rb

```diff
@@ -178,6 +190,15 @@ def self.find_id(to_find)
 
   private
 
+  def safe_delete
+    pglogical.drop_subscription(id, true)
+  rescue PG::InternalError => e
+    raise unless e.message =~ /could not connect to publisher/ || e.message =~ /replication slot .* does not exist/
```

Comment:
If someone has a non-English postgres installation, will this still work?

Reply:
Maybe not? I can try to change the appliance locale and see if things break, but the alternative would be to just remove this conditional and allow the next command to raise if it was some other error so I feel like it makes sense to just leave it in given that we don't have a more specific error to rescue.

Reply:
Ok.  I didn't know if it was a problem if the code tries to move forward or not.  I know in the past that message text checking has been brittle in particular with non-English databases, but I assume this was last resort for you because there wasn't something more specific.